.gitignore

```diff
@@ -0,0 +1,5 @@
+# Ignore IntelliJ files
+*.iml
+.idea
+
```

Comment:
IMHO, this should be added in your `info/exclude` file, as there may be developers not interested in IDE configs. What do you think? https://blog.martinhujer.cz/dont-put-idea-vscode-directories-to-projects-gitignore/

Reply:
Sure thing, whatever you think is cleaner.

On a sidenote - wow that article is savage! :fire: 

> On the other hand, do you expect to receive high-quality contributions from developers who don't bother to properly configure their workstations?